roles/openshift_node/templates/atomic-openshift-node.service.j2

```diff
@@ -4,11 +4,17 @@ After={{ openshift.docker.service_name }}.service
 After=openvswitch.service
 Wants={{ openshift.docker.service_name }}.service
 Documentation=https://github.com/openshift/origin
+Requires=dnsmasq.service
+After=dnsmasq.service
 
 [Service]
 Type=notify
 EnvironmentFile=/etc/sysconfig/atomic-openshift-node
 Environment=GOTRACEBACK=crash
+ExecStartPre=/usr/bin/cp /etc/origin/node/node-dnsmasq.conf /etc/dnsmasq.d/
```

Comment:
*edit* I had a comment here asking why this was necessary, and then I saw the explanation in `roles/openshift_node_dnsmasq/tasks/main.yml`